libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1405,6 +1408,18 @@ AP_GPS_UBLOX::_parse_gps(void)
         state.location.lng    = _buffer.pvt.lon;
         state.location.lat    = _buffer.pvt.lat;
         state.location.alt    = _buffer.pvt.h_msl / 10;
+        // Check if valid fix
+        if (!(_buffer.pvt.flags & 0b00000001) 
```

Comment:
Is there a doc defining which flag correlates to which bit within the bitmap that I can reference?

Reply:
https://cdn.sparkfun.com/assets/learn_tutorials/1/1/7/2/ZED-F9R_Interfacedescription__UBX-19056845_.pdf